src/components/calcite-alert/readme.md

```diff
@@ -47,7 +46,7 @@ Single alert for toast notifications. To position the alert correctly and manage
 
 ### `closeCalciteAlert() => Promise<void>`
 
-emit the `calciteAlerClose` event - <calcite-alerts> listens for this, if the alert is not active, but is the queue, this will remove it from the queue
+emit the `calciteAlerClose` event - <calcite-alerts> listens for this
```

Comment:
`calciteAlerClose` -> `calciteAlertClose`.

Reply:
This is mine. Will fix.